packages/vue/src/StoreDesign/RequestForm/RequestForm.vue

```diff
@@ -270,8 +270,8 @@ const toggleSubForm = (name) => {
       :preview-mode="previewMode"
       @on-update="onUpdateForm"
     >
-      <template #SelectProducts>
-        <slot name="SelectProducts"></slot>
+      <template #SelectProducts="slot2">
```

Comment:
can we use a more meaningful name other than `slot2`? something more descriptive and contextual. it would make it easy for someone else to grasp what you are trying to do here.